iree/tools/iree-e2e-matmul-test.c

```diff
@@ -174,6 +175,7 @@ static iree_status_t reference_matmul(iree_vm_list_t* input_list,
   iree_hal_dim_t m_size, k_size, n_size;
   IREE_RETURN_IF_ERROR(
       get_matmul_sizes(lhs, rhs, acc, result, &m_size, &k_size, &n_size));
+  fprintf(stderr, "Matmul shape (MxKxN): %dx%dx%d\n", m_size, k_size, n_size);
```

Comment:
No, I was really missing this information in the logs. That's because we are so far only logging the function name, because that's the only thing that's easy to access (without substantial new code just for that ) at the place where we're currently doing the logging (which is the right place, and moving that to a more internal place where more things are at hand would risk mis-logging).

Reply:
can you just name the functions with the MxKxN as you're the one generating them? that gives you something you can track across the whole compiler/runtime flow